src/TypeScript/EslintBridgeClient/Contract/AnalysisResponse.cs

```diff
@@ -81,6 +82,9 @@ internal class Issue
 
         [JsonProperty("cost")]
         public int? Cost { get; set; }
+
+        [JsonProperty("quickFixes")]
+        public QuickFix[] QuickFixes { get; set; }
```

Comment:
Support for quick fixes. For now I re-defined the types that are in core so that I could put JsonPropertyAttribute on them. We should think how we want to handle it. I don't think it makes sense for the types in core to have attributes with values that are specific to eslint-bridge.